common/static/common/js/karma.common.conf.js

```diff
@@ -330,7 +337,11 @@ var getBaseConfig = function (config, useRequireJs) {
         // how many browser should be started simultaneous
         concurrency: Infinity,
 
-        browserNoActivityTimeout: 50000
+        browserNoActivityTimeout: 50000,
+
+        client: {
+            captureConsole: false
```

Comment:
I'm nervous that we'll have difficulty debugging in the future without this information being logged. Is this only an issue on Firefox? Can we only disable it for Firefox if that's the case? 

I'm assuming that this PR is to fix the console output like this that we've started seeing in the logs:

```
19:09:59 ERROR LOG: '@http://localhost:9876/base/common/js/vendor/jquery.js?26e6b63b81813d8ad942c90d369df2673602b812 line 343 > eval:1:1
19:09:59 .globalEval@http://localhost:9876/base/common/js/vendor/jquery.js?26e6b63b81813d8ad942c90d369df2673602b812:343:5
19:09:59 domManip@http://localhost:9876/base/common/js/vendor/jquery.js?26e6b63b81813d8ad942c90d369df2673602b812:5291:1
19:09:59 .append@http://localhost:9876/base/common/js/vendor/jquery.js?26e6b63b81813d8ad942c90d369df2673602b812:5431:1
19:09:59 .html/<@http://localhost:9876/base/common/js/vendor/jquery.js?26e6b63b81813d8ad942c90d369df2673602b812:5525:5
19:09:59 access@http://localhost:9876/base/common/js/vendor/jquery.js?26e6b63b81813d8ad942c90d369df2673602b812:3614:5
19:09:59 .html@http://localhost:9876/base/common/js/vendor/jquery.js?26e6b63b81813d8ad942c90d369df2673602b812:5492:1
19:09:59 XBlockView<.updateHtml@http://localhost:9876/base/js/views/xblock.js?6e4957101b5ba5624a447c26c774228a803b70bb:9:6417
```

I would be interested in understanding what is being reported before we hide it. Maybe it is indicative of a real problem that we can fix, and then the size of the console will be less of an issue.